platform/core/Makefile

```diff
@@ -1,30 +1,75 @@
 .DEFAULT_GOAL := all
 
+build_type := $(if $(filter $(MODE), release),release,debug)
+release_flag := $(if $(filter $(build_type), release), --release)
+target := target/$(build_type)
+
+# iOS export
 export_dir_ios := export/ios
 swift_core := PolyPodCoreSwift
 swift_core_framework := $(swift_core)/PolyPodCore.xcframework
 ios_archs := aarch64-apple-ios aarch64-apple-ios-sim x86_64-apple-ios
 ios_targets := iPhone simulator
-build_type := $(if $(filter $(MODE), release),release,debug)
 
-ifneq ($(OS),Windows_NT)
+# Flatbuffer tools
+flatbuffers_version := v2.0.0
+flatbuffers_repo := https://github.com/google/flatbuffers
+flatbuffers_releases := $(flatbuffers_repo)/releases/download/$(flatbuffers_version)
+flatbuffers := flatbuffers_shared/flatbuffers_$(flatbuffers_version)
+flatc_dir := flatbuffers_shared/flatc_$(flatbuffers_version)
+flatc := $(flatc_dir)/flatc
+
+# Flatbuffer models
+flatbuffer_models_dir := flatbuffers_shared/flatbuffer_models
+flatbuffer_models := $(shell find $(flatbuffer_models_dir) -name '*.fbs')
+rust_flatbuffers_dir := src/flatbuffers_generated
+rust_flatbuffer_models = $(patsubst $(flatbuffer_models_dir)/%, $(rust_flatbuffers_dir)/%, $(flatbuffer_models:.fbs=.rs))
+rust_flatbuffers_generated_lib := $(rust_flatbuffers_dir)/mod.rs
+swift_flatbuffers_dir := PolyPodCoreSwift/Sources/PolyPodCoreSwift/FlatbuffersGenerated
+swift_flatbuffer_models = $(patsubst $(flatbuffer_models_dir)/%, $(swift_flatbuffers_dir)/%, $(flatbuffer_models:.fbs=.swift))
+
+ifeq ($(OS),Windows_NT)
+	flatc_download_url = $(flatbuffers_releases)/Windows.flatc.binary.zip
+	flatc := $(flatc_dir)/flatc.exe
+	# Need on Windows, to allow interpreting escaping characters
+	os_echo := echo -e
+else
+	flatc := $(flatc_dir)/flatc
+	os_echo := echo
 	ifeq ($(shell uname -s),Darwin)
+		flatc_download_url = $(flatbuffers_releases)/Mac.flatc.binary.zip
 		macos := true
 	endif
+	ifeq ($(shell uname -s),Linux)
+		flatc_download_url = $(flatbuffers_releases)/Linux.flatc.binary.clang++-9.zip
+	endif
 endif
 
-.PHONY: flatbuffers
-flatbuffers:
-	./flatbuffers_shared/setup_flatbuffers.sh
-	./flatbuffers_shared/generate_flatbuffers.sh
+$(flatbuffers):
+	git clone --branch $(flatbuffers_version) $(flatbuffers_repo) $(flatbuffers)
 
-target: flatbuffers
-	cargo build
+$(flatc):
+	mkdir -p $(flatc_dir)
+	curl -L $(flatc_download_url) -o $(flatc_dir)/flatc.zip && unzip -d $(flatc_dir) $(flatc_dir)/flatc.zip && rm $(flatc_dir)/flatc.zip
+	chmod +x $(flatc)
 
-$(swift_core_framework): target
+$(rust_flatbuffers_dir)/%.rs: $(flatbuffer_models_dir)/%.fbs
+	./$(flatc) --rust -o $(rust_flatbuffers_dir) --filename-suffix "" --include-prefix flatbuffers_generated $^
+
+$(swift_flatbuffers_dir)/%.swift: $(flatbuffer_models_dir)/%.fbs
+	./$(flatc) --swift -o $(swift_flatbuffers_dir) --filename-suffix "" $^
+
+$(rust_flatbuffers_generated_lib): $(rust_flatbuffer_models)
+	$(shell $(os_echo) $(foreach model, $^, \
```

Comment:
Wouldn't this command run in a shell even if you don't do this? This is generally done if you want to assign the output to a variable or something like that.

Reply:
You are right, looks to be a testing leftover.